seatunnel-engine/seatunnel-engine-core/src/main/java/org/apache/seatunnel/engine/core/dag/logical/LogicalDagGenerator.java

```diff
@@ -81,13 +81,13 @@ private void createLogicalVertex(Action action) {
 
     private Set<LogicalEdge> createLogicalEdges() {
         return inputVerticesMap.entrySet()
-            .stream()
-            .map(entry -> entry.getValue()
-                    .stream()
-                    .map(targetId -> new LogicalEdge(logicalVertexMap.get(entry.getKey()),
-                        logicalVertexMap.get(targetId)))
-                    .collect(Collectors.toList()))
-            .flatMap(Collection::stream)
-            .collect(Collectors.toSet());
+                .stream()
```

Comment:
It seems you and @ashulin  are try to solve a same problem？

Reply:
Just use to test, and update by mistake.